rllib/models/tf/tf_action_dist.py

```diff
@@ -97,6 +97,16 @@ def required_model_output_shape(action_space, model_config):
         return action_space.n
 
 
+def get_categorical_with_temperature(t: float):
```

Comment:
nit: Can we update the name to indiciate this is returning a class? get_categorical_class_with_temperature?

Reply:
done